src/meta.rs

```diff
@@ -48,8 +56,12 @@ impl<E: EthSpec> Multiplexer<E> {
     pub async fn proxy_directly(&self, request: Request) -> Result<Response, ErrorResponse> {
         let id = request.id;
 
-        // TODO: adjust timeout
```

Comment:
I think a dynamic timeout configurable via the CLI would be better. Currently your PR just adds some different hardcoded constants for other requests

Do you want to try adding a CLI parameter? It could work similarly to Lighthouse's `--execution-timeout-multiplier`

Reply:
I'll convert this one into CLI parameter.